amundsen_application/static/js/components/common/ResourceList/index.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import Pagination from 'react-js-pagination';
+import ResourceListItem from 'components/common/ResourceListItem';
+import { Resource } from 'components/common/ResourceListItem/types';
+import { ITEMS_PER_PAGE, PAGINATION_PAGE_RANGE } from "./constants";
+
+
+export interface ResourceListProps {
+  activePage: number;
```

Comment:
One thing I've been mulling about was where to keep track of the `activePage` state. In some cases it needs to be managed externally (SearchPage) and in others it can managed internally in the `ResourceList` component (MyBookmarks).

I could make the `activePage` and `onPagination` props as optional. Any thoughts on this?

Reply:
Yeah, I think I'll make all of the pagination stuff optional in addition to providing some default state management. Will push a PR shortly.